elasticsearch/sgconfig/roles.yml

```diff
@@ -168,13 +135,20 @@ project_user:
   readonly: true
   cluster:
     - CLUSTER_COMPOSITE_OPS_RO
+    - indices:data/write/bulk  #required for being able to let index mappings update... is this required still with multitenancy?
   indices:
     app:
       '*':
         - READ
-      _dls_: "{\"bool\":{\"filter\":{\"script\":{\"script\":{\"lang\":\"painless\",\"params\":{\"param1\":\"${attr.proxy.ns}\"},\"source\":\"String namespace = doc['kubernetes.namespace_name.keyword'][0];StringTokenizer st = new StringTokenizer(params.param1,\\\",\\\");while (st.hasMoreTokens()){if (st.nextToken().equalsIgnoreCase(namespace)){return true;}}return false;\"}}}}}"
-    logs-app:
```

Comment:
multi-tenancy test is failing because the `logs-app` alias is removed from index patterns.

The test adds `logs-app` alias to the created index, and not both `app` and `logs-app`. So if possible can you please modify the test case to add both alias? [here](https://github.com/openshift/origin-aggregated-logging/blob/master/test/multi_tenancy.sh#L111)
Thx!!

@jcantrill

Reply:
or let me know i can send a tiny PR for this change ahead of this PR merging

Reply:
@vimalk78 i will fix the tests -- we don't need both 'app' and 'logs-app' for it to work in kibana.